src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -126,9 +130,12 @@ IObservable<Unit> OnCloneRepository(object state)
                     notificationService.ShowError(Resources.RepositoryCloneFailedNoSelectedRepo);
                     return Observable.Return(Unit.Default);
                 }
+                
                 // The following is a noop if the directory already exists.
                 operatingSystem.Directory.CreateDirectory(BaseRepositoryPath);
-                return cloneService.CloneRepository(repository.CloneUrl, repository.Name, BaseRepositoryPath);
+
+                return cloneService.CloneRepository(repository.CloneUrl, repository.Name, BaseRepositoryPath)
+                    .Do(_ => this.usageTracker.IncrementCloneCount());
```

Comment:
True... do we validate if the target clone path already exists? That's the only error that I can see us preventing.

EDIT: Nope, we don't, so users can potentially select a base path to clone into and VS fails because there's already a directory with the same name in there. Feels like something we should fix 😜